### PR TITLE
Tech Trader - READY FOR MERGE

### PR DESCRIPTION
**Pull request by mharris717**
Don't merge until the Business First cards spoiled on 2/26 are in nrdb. 

Seemed pretty idiot proof since I could practically copy Geist, but those are famous last words

### CODE REVIEW

File: src/clj/game/cards-resources.clj [hunk 1]
```diff
@@ -759,6 +759,11 @@
                                :no-ability {:effect (effect (clear-wait-prompt :corp))}}}
                             card nil))}}}
 
+   "Tech Trader"
+   {:events {:runner-trash {:req (req (and (= side :runner) (= (second targets) :ability-cost)))
+                            :msg "Gain a credit"
```
Comment (nealterrell) [inline src/clj/game/cards-resources.clj]: Remember that `:msg` is always shown as "PlayerName uses CardName to ... [msg]", so this should be `"gain 1 [Credit]"`.

Comment (mharris717) [inline src/clj/game/cards-resources.clj]: Crap sorry, had a placeholder message and forgot to change it

**Comment by mtgred** [comment]
The Business First is now available on NetrunnerDB.

A Circle CI check seems to fail. Can you have a look?

**Comment by mharris717** [comment]
@mtgred just needed to rerun since cards got added, passes now

**Comment by mtgred** [comment]
:+1:

**Pull request merged**